docs/assembly.rst

```diff
@@ -333,11 +333,12 @@ push their entry label (with virtual function resolution applied). The calling s
 in solidity are:
 
  - the caller pushes return label, arg1, arg2, ..., argn
```

Comment:
Exactly - the given names of stack slots is the exact stack layout at its top and it changes from the layout in the first line to the layout in the second line. Will clarify.